config/settings/base.py

```diff
@@ -85,6 +84,7 @@
 
 LOCAL_APPS = [
     "fv_be.users",
+    "fv_be.app",
```

Comment:
Small change to base configuration to fix errors when unregistering models on the admin site